### ISSUE TITLE
About the spilt of train and test dataset

### ISSUE THREAD

**Issue by MontaEllis** [eladrich/pixel2style2pixel#233]
Great work! But I have one simple question. You train on FFHQ and test on the test set of CelebA-HQ.  What about CelebA-HQ's training set? Did you use it?

**Comment by yuval-alaluf** [comment]
For the inversion task we did not use the CelebA-HQ train set

**Comment by MontaEllis** [comment]
Thanks a lot! How you spilt train and test dataset?

**Comment by yuval-alaluf** [comment]
Using the official data preprocessing script: https://github.com/switchablenorms/CelebAMask-HQ/tree/master/face_parsing

**Comment by MontaEllis** [comment]
Thanks! So you just use 2824 imgs for test?

**Comment by yuval-alaluf** [comment]
Yes
